internal/command/junit/junit_internal_test.go

```diff
@@ -151,3 +214,21 @@ func Test_suiteFilesAsSortedList(t *testing.T) {
 		})
 	}
 }
+
+// From internal/backend/remote-state/s3/testing_test.go
+// diagnosticComparer is a Comparer function for use with cmp.Diff to compare two tfdiags.Diagnostic values
```

Comment:
Should/could this be moved into the tfdiags package? Or are there other ways that diagnostics are compared in the code base that are a better option?

Reply:
I think moving it into `tfdiags` is a good idea!

We could follow the pattern of ctydebug - https://github.com/zclconf/go-cty-debug/blob/0d6042c539401a57fc0cca85ded2861d4a5173c4/ctydebug/cmp.go#L19-L42 I don't mean the implementation necessarily but more the fact that it exports a `cmp.Option` variable which can be _directly_ used as an argument in `cmp.Diff`.